.github/workflows/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug report
+about: Report an issue to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+Please describe the issue.
+
+**Your environment**
+- `fal` version: 
+- OS:
+- Other relevant dependencies & versions:
+
```

Comment:
Could we change this to:

~~~suggestion
**Your environment**
- OS: 
- Paste the following commands output:
  ```
  fal --version
  dbt --version
  ```
- Adapter being used: <!-- (if more than one dbt plugin installed) --> 

~~~

To get more information needed to reproduce.

Reply:
That's a good idea, thanks!